### ISSUE TITLE
[3.2] Roll-over enemies

### ISSUE THREAD

**Issue by MuadDib90** [Realm667/WolfenDoom#1228]
After recent playthroughs I must say for myself, personally, it's getting (sick) frustrating of roll-over enemies.

As @Ozymandias81 stated, they are not "invincible" in this state, but rather less prone to damage. But that is the issue. If you get two direct hits from shotgun from close range, and they are still not dead, that is... well, not good. (They are roll.-overing all of the time and you are just wasting your ammo,)

Is there somethig to have some work-around of this issue, or is this hitting the limitations of the engine and there is nothing to be done to get rid of this, please?

@Tormentor667 @AFADoomer @Talon1024

**Comment by Tormentor667** [comment]
In which skill setting did you play?

**Comment by MuadDib90** [comment]
The fourth - Skill 3
(Is it from 0 to 4, right?)

**Comment by AFADoomer** [comment]
The reduced damage originated from when sprites weren't getting properly set when an enemy was damaged or killed while rolling.  They originally took no damage at all while rolling; that was changed to 50% (regardless of skill level) around 3 years ago.  

Damage amount can be further reduced without breaking anything technically - I'd recommend randomizing it and/or tying it to skill level.

**Comment by MuadDib90** [comment]
Well, randomizing it sounds great; like the probability of dodging the bullet.
How about to set the reduction from 0 % to 50 % for example? (At random cycle.)
Also, I don't think it needs to be skill dependent, there are already rules for each of them.
What do you think?

**Comment by MuadDib90** [comment]
Last weekend I played C2M6 level and guess what happend?

I burned all fuel (50 units) of flamethrower on that one strong soldier with mask down the facility (which he is not supposed to survive), but he did. It was because when he is hit, he roll-overs; and when he "is being hit" he roll-over constantly. So what happend: he was cornered (literally in corner), I was shooting a flamethrower on him and he was just roll-overing again and again and again... until I run out of fuel, and he got me.

This is (for me at least) a nice example of how the reduced demage, when in roll-over state, gets reeeally annoying.
